src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -249,6 +250,21 @@ public void UseShellExecute_GetSetWindows_Success()
             Assert.False(psi.UseShellExecute, "UseShellExecute=true is not supported on onecore.");
         }
 
+        [PlatformSpecific(TestPlatforms.Windows)]
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework, "Desktop UseShellExecute is set to true by default but UseShellExecute=true is not supported on Core")]
+        public void UseShellExecute_GetSetWindows_Success_Netfx()
+        {
+            ProcessStartInfo psi = new ProcessStartInfo();
+            Assert.True(psi.UseShellExecute);
+
```

Comment:
It's not a goal to create a test suite for desktop. If we ever have tests specifically for desktop, they are more for documenting some difference, if anything.